### PR TITLE
Refactor Creation of Headless Image

### LINKED ISSUE #123: Server in the second image
As a <kbd>User</kbd> I want to have a server within the second image in order to *communicate with it*.

### Acceptance criteria

- [x] A server runs in the second image
- [x] Accepts requests and returns responses
- [x] The server stops whenever the second image is closed

### Effort Evaluation

6 Points

### PR DESCRIPTION
**Pull request by benjamin-frost**
Closes: #123

### CODE REVIEW

File: packages/AutoTDD-Core.package/ATDDHeadlessImage.class/instance/process..st [hunk 1]
```diff
@@ -0,0 +1,3 @@
+accessing
+process: aProcess
+	process := aProcess
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDHeadlessImage.class/instance/process..st]:
```suggestion
accessing
process: aProcess

	process := aProcess
```

File: packages/AutoTDD-Core.package/ATDDHeadlessImage.class/instance/process.st [hunk 2]
```diff
@@ -0,0 +1,3 @@
+accessing
+process
+	^ process
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDHeadlessImage.class/instance/process.st]:
```suggestion
accessing
process

	^ process
```

**Review by SebastianWagner2 — CHANGES_REQUESTED** [review]
add newLines (conversations), add class comment (for ATDDHeadlessImage)

**Review by kolioOtSofia — APPROVED** [review]

**Review by Toalaah — APPROVED** [review]

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/start.st [hunk 3]
```diff
@@ -0,0 +1,6 @@
+initialization
+start
+
+	FileDirectory default forceNewFileNamed: 'startServerScript.st' do: [ :str | str nextPutAll: 'ATDDHeadlessImage startServer.'].
```
Comment (Toalaah) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/class/start.st:4]:
```suggestion
	FileDirectory default forceNewFileNamed: 'startServerScript.st' do: [ :str | str nextPutAll: 'ATDDTestingImage startServer.'].
```

Comment (Toalaah) [reply packages/AutoTDD-Core.package/ATDDTestingImage.class/class/start.st:4]: I think you forgot to rename the class here, probably didn't do it automatically.

**Review by Toalaah — CHANGES_REQUESTED** [review]

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/start.st (hunk 3 above)
Comment (benjamin-frost) [reply packages/AutoTDD-Core.package/ATDDTestingImage.class/class/start.st:4]: Good point :+1: That would have been bad :D

**Review by Toalaah — APPROVED** [review]

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+An ATDDTestingImage is a second image that is being opened in order to run tests in parallel. To make this seamless to the user the image is opened headless.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/README.md]:
```suggestion
An ATDDTestingImage is a second image that is being opened in order to run tests in parallel. To make this seamless to the user the image is opened headlessly.
```

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process..st [hunk 5]
```diff
@@ -0,0 +1,4 @@
+accessing
+process: anOSProcess
+
+	Process := anOSProcess
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process..st]:
```suggestion
accessing
process: anOSProcess

	process := anOSProcess
```
Don't use class variable names, use instance variables on class side instead (and start with small letter)

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process.st [hunk 6]
```diff
@@ -0,0 +1,4 @@
+accessing
+process
+
+	^ Process
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process.st]:
```suggestion
accessing
process

	^ process
```

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/startServer.st [hunk 7]
```diff
@@ -0,0 +1,9 @@
+initialization
+startServer
+
+	| server |
+	
+	server := WebServer new.
+	server listenOn: 8888;
+		addService: '/' action: [ :req | req send200Response: 'hello World'].
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/class/startServer.st]:
```suggestion
		addService: '/' action: [ :req | req send200Response: 'Hello World'].
```

**Review by SebastianWagner2 — CHANGES_REQUESTED** [review]

**Review by SebastianWagner2 — APPROVED** [review]

**Pull request merged**